Assets/Scripts/Combat/Combantants/Combatant.cs

```diff
@@ -43,9 +47,17 @@ public void Update()
 
     private void UpdateHealthBar()
     {
-        string healthText = IsAlive ?
-            $"{CurrentHealthPoints} / {MaxHealthPoints}" :
-            "I've fallen and can't get up";
+        string healthText;
+
+        if (CurrentShieldHealth > 0)
+            healthText = IsAlive ?
+                        $"{CurrentHealthPoints} / {MaxHealthPoints} Shield: {CurrentShieldHealth} / {MaxShieldHealth}" :
+                        "I've fallen and can't get up";
+
+        else
+            healthText = IsAlive ?
+                $"{CurrentHealthPoints} / {MaxHealthPoints}" :
+                "I've fallen and can't get up";
 
         Vector3 relativeScreenPosition = Camera.main.WorldToScreenPoint(transform.position);
         relativeScreenPosition.y += 125f;
```

Comment:
Need to change this hardcoded value

Reply:
This was already in the script when I was making the ability, but I'll change it